docs/components/zeebe/zeebe-overview.md

```diff
@@ -5,21 +5,25 @@ sidebar_label: "Introduction"
 description: "Zeebe is the workflow engine powering Camunda 8. While written in Java, you do not need to be a Java developer to use Zeebe."
 ---
 
-Zeebe is the workflow engine powering Camunda 8. While written in Java, you do not need to be a Java developer to use Zeebe.
+import OverviewImg from '../zeebe/technical-concepts/assets/zeebe-architecture.png';
+import ZeebeGrid from '../zeebe/react-components/\_zeebe-card';
 
-With Zeebe you can:
+import { gettingStartedCards } from '../zeebe/react-components/\_zeebe-card-data';
 
-- Define processes graphically in [BPMN 2.0](../modeler/bpmn/bpmn-coverage.md).
-- Choose any [gRPC](/apis-tools/zeebe-api/grpc.md)-supported programming language to implement your workers.
-- Build processes that react to events from Apache Kafka and other messaging platforms.
-- Use as part of a software as a service (SaaS) offering with Camunda 8 or deploy with Docker and Kubernetes (in the cloud or on-premises) with Camunda 8 Self-Managed.
-- Scale horizontally to handle very high throughput.
-- Rely on fault tolerance and high availability for your processes.
-- Export processes data for monitoring and analysis (available through the [exporters](../../self-managed/zeebe-deployment/exporters/exporters.md) added in Camunda 8 Self-Managed).
-- Engage with an active community.
+<p><a title="Zeebe architecture" href="../technical-concepts/architecture"><img src={OverviewImg} alt="Zeebe architecture" style={{border:0,padding:0,paddingLeft:20,margin:0,float: 'right', width: '60%'}} className="fade-in-top-image"/></a>Zeebe is the process automation engine powering Camunda 8. While written in Java, you do not need to be a Java developer to use Zeebe.</p>
+
+A workflow engine is an essential part of any process automation tool. We call it an “engine” because it drives business processes from start to finish, no matter how complex the process and decision logic need to be.
+
+## Why Zeebe?
+
+Zeebe doesn’t rely on a central database, so there’s no performance bottleneck as process volumes increase. Deliver high throughput by distributing processing across clusters, or add cluster nodes to execute an unlimited number of processes at consistently low latency.
+
+Zeebe distributes data across all brokers in a cluster with storage directly on the server filesystem. If one broker goes down, another can replace it with no data loss. This pre-configured replication mechanism ensures that Camunda can recover from machine or software failure with no human interaction, no data loss, and minimal downtime.
 
 For documentation on deploying Zeebe as part of Camunda 8 Self-Managed, refer to the [deployment guide](../../self-managed/zeebe-deployment/zeebe-installation.md).
 
-## Next steps
+## Get started
+
+New to Zeebe? Learn about clustering, partitions, internal processing, and more.
 
-- Get familiar with [technical concepts](technical-concepts/technical-concepts-overview.md).
+<ZeebeGrid zeebe={gettingStartedCards} />
```

Comment:
Looks great. 🚀 I guess it is expected that the links don't work in the preview? ❓

Reply:
This works now 👍